pkg/vminfo/linux.go

```diff
@@ -44,11 +44,6 @@ func (linux) machineInfos() []machineInfoFunc {
 }
 
 func (linux) parseModules(files filesystem) ([]*KernelModule, error) {
-	_, err := files.ReadFile("/proc/sentry-meminfo")
```

Comment:
Please also remove "sentry-meminfo" from the array above.

Reply:
Done.